src/gsi/setuprad.f90

```diff
@@ -1298,11 +1298,11 @@ subroutine setuprad(obsLL,odiagLL,lunin,mype,aivals,stats,nchanl,nreal,nobs,&
               do i=1,nchanl
                 pred(6,i) = zero
                 pred(7,i) = zero
-                clw_avg = half*(clw_obs+clw_guess_retrieval)
+                clw_avg = half*(clw_obs+clw_guess_retrieval) * zero
```

Comment:
Since we multiply `half*(clw_obs+clw_guess_retrieval)` by `zero`, why don't we just set `clw_avg = zero`?

Reply:
The cloud's impacts are still need to be investigated further, though we set it to zero now.  Therefore we want to keep the formula (or the piece of codes) as it is in the file.

Reply:
Thanks for the explanation. My preference is to not see extra computations which eventually wind up being zero.  Code is version controlled so we can always see code from previous versions commits.  Could also rewrite as
```
! Need to investigate clw_avg = half*(clw_obs+clw_guess_retrieval) * zero
clw_avg = zero
```

Reply:
Changed, thanks.